src/main/java/UserInterface/GUI/GUIPages/RecipeBook.java

```diff
@@ -4,13 +4,10 @@
 import main.java.UserInterface.Commands.RecipeBookCommands.RemoveRecipeCommand;
```

Comment:
There is an updated file in Dohyun's pull request I believe. So these changes need to be transferred to that file.